.github/workflows/version-snapshot.yml

```diff
@@ -0,0 +1,45 @@
+name: Version Snapshot Package
+
+on:
+  push:
+    branches:
+      - 'snapshot*'
+
```

Comment:
how are we making sure this doesn't get into loop. Like snapshot changeset we push it to this snapshot branch -> runs `pnpm ci:prerelease` creates a PR with deletion of changeset and updates version to `0.0.0-.....` gets pushed to snapshot branch...doesn't this run again with `pnpm ci:prerelease` ?

Reply:
Yeah makes sense